department-of-veterans-affairs/va.gov-cms
# VACMS-4936: Add setter logic to vc ief selects.

## Description
See #4936

## Testing done
Visual

## Screenshots
![image](https://user-images.githubusercontent.com/2404547/123164461-f9455c80-d440-11eb-836a-6ffc66df345e.png)

## QA steps
- As an administrator, go to `node/3756/edit` and add a service via the inline form.
- Visually verify the admin and office fields are populated with Escanaba.
- Save the node, and verify that the service and entity refs are preserved.

### Definition of Done


### Select Team for PR review


### Is this PR blocked by another PR?


### Does this PR need review from a Product Owner


### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
